ti/config/config.go

```diff
@@ -22,6 +23,7 @@ func New(endpoint, token, accountID, orgID, projectID, pipelineID, buildID, stag
 		commitBranch: commitBranch,
 		dataDir:      dataDir,
 		ignoreInstr:  false,
+		parseSavings: parseSavings,
```

Comment:
Can we pass a ff map instead of each ff as a separate argument? That way we wont have to keep updating the argument list when we add or deprecate ffs. What do the others think about this? @hemanthmantri @jamie-harness

Reply:
IMO step interface should be independent of any feature flags. Adding FF map here would just make it error prone